src/main/java/com/google/devtools/build/lib/remote/util/Utils.java

```diff
@@ -107,6 +107,16 @@ public static boolean hasTopLevelOutputs(
     return !Collections.disjoint(outputs, topLevelOutputs);
   }
 
+  public static String grpcAwareErrorMessage(IOException e) {
```

Comment:
How about a few simple tests for this method?

Reply:
done